elastic/beats
# [Heartbeat] Add keyword mapping for service.name

Fixes https://github.com/elastic/uptime/issues/452 by adding the proper mapping for this field that was introduced some time ago

## Checklist




## How to test this PR locally

Delete all `heartbeat*` indices, run some monitors and check that the following aggregation works:

```
GET heartbeat*/_search
{
  "size": 0,
  "aggs": {
    "s": {"terms": {"field": "service.name"}}
  }
}
```
